pkg/pillar/devicenetwork/dnc.go

```diff
@@ -734,18 +736,7 @@ func checkAndUpdateWireless(ctx *DeviceNetworkContext, oCfg *types.DevicePortCon
 				}
 			}
 		}
-		if oldPortCfg == nil || strings.Contains(portCfg.Key, "override") ||
-			!reflect.DeepEqual(oldPortCfg.WirelessCfg, pCfg.WirelessCfg) {
-			// XXX hack to disable the cloud configure for wifi, if the existing configure
-			// has wifi information, but the new update does not. Remove this when the zedcloud
-			// implements the wifi configure for port
-			if oldPortCfg != nil && oldPortCfg.WirelessCfg.WType == types.WirelessTypeWifi &&
-				pCfg.WirelessCfg.WType == types.WirelessTypeNone {
-				log.Infof("checkAndUpdateWireless: HACK! wifi has configure %+v, ignore the update %+v\n",
-					oldPortCfg.WirelessCfg, pCfg.WirelessCfg)
-				break
-			}
-			// XXX
+		if oldPortCfg == nil || isOverride || !reflect.DeepEqual(oldPortCfg.WirelessCfg, pCfg.WirelessCfg) {
```

Comment:
Once controller supports WiFi config, shouldn't the 'isOverride' check go away here as well? Or the whole comparison of WirelessCfg should go away? 
Please add comment indicating what should be removed once we have controller support.